client/components/QuestionsAndAnswers/modal/QuestionForm.jsx

```diff
@@ -25,7 +25,7 @@ const QuestionForm = ({
           id="question"
           type="text"
           maxlength="1000"
-          onChange={(e) => setNewQuestion({ ...newQuestion, question_body: e.target.value })}
+          onChange={(e) => setNewQuestion({ ...newQuestion, body: e.target.value })}
```

Comment:
If this was added for form submission, I think an onSubmit on the form tag would work better! That way you can access all of the inputs through `e.target.YourHtmlFor`. I think it's not a good idea to call `setState` with each keystroke. It might useful if you needed to keep track of the state of the new question for this product if you exit the modal view and want to have that same question when you enter modal view again.